lesson07/assignment/cli3/src/views/Images.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+<div class="home">
+  <div class="container">
+    <div class="container bg-info text-white">
+      <h2>Under the Sea:</h2>
+      <!--Below 993px, resizes image to match screensize-->
+
+      <img class="img-fluid" alt="a ray beneath the waves" src="../assets/ray.png">
+    </div>
+    <div class="container bg-primary text-white">
+      <h2>In Rome:</h2>
+      <!--Below 993px, resizes image to match screensize-->
+      <img class="img-fluid" alt="the Roman Colosseum" src="../assets/rome.png">
+    </div>
+  </div>
+
+</div>
+</template>
+
+<script>
+// @ is an alias to /src
```

Comment:
You'll want to add in the image's alt/src in the data and use v-for to loop over it.